pyscript.core/src/core.js

```diff
@@ -218,16 +218,28 @@ class PyScriptElement extends HTMLElement {
     set id(value) {
         super.id = value;
     }
+
+    get srcCode(){
+        console.warn(
+            `'srcCode' Attribute is Deprecated: use the 'textContent' attribute to retrieve code from a <py-script> or <script type='py'> tag.`,
+        );
+        return this._srcCode;
+    }
+
+    set srcCode(code){
```

Comment:
to be honest I don't think this was ever meant to be possible ... the `srcCode` is the initial code that run into the node, I am not sure why anyone would change that at runtime :thinking: maybe we should remove the setter?